OrbitGl/App.cpp

```diff
@@ -637,13 +636,26 @@ ErrorMessageOr<void> OrbitApp::OnSaveCapture(const std::string& file_name) {
                                   timers_it_end);
 }
 
-ErrorMessageOr<void> OrbitApp::OnLoadCapture(const std::string& file_name) {
+void OrbitApp::OnLoadCapture(const std::string& file_name) {
+  if (open_capture_callback_) {
+    open_capture_callback_();
+  }
   ClearCapture();
+  GCurrentTimeGraph->GetStringManager()->Clear();
```

Comment:
Should the `StringManager` be accessed from `GOrbitApp`, which is its actual owner?

Reply:
Done. Thanks